addons/binding/org.openhab.binding.pentair/src/main/java/org/openhab/binding/pentair/handler/PentairEasyTouchHandler.java

```diff
@@ -188,7 +188,7 @@ public void processPacketFrom(PentairPacket p) {
                 break;
             case 2:
                 if (p.getLength() != 29) {
-                    logger.debug("Expected length of 29: ", p);
+                    logger.debug("Expected length of 29: {}", p);
```

Comment:
Does have a proper toString?

Reply:
The toString method of PentairPacket returns a hex String.